lib/entry.js

```diff
@@ -94,100 +80,42 @@ Ep.runModuleGetters = function (module) {
   }
 };
 
-// Returns true iff the getter updated module.exports with a new value.
 Ep.runGetter = function (module, name) {
-  if (hasOwn.call(this.getters, name)) {
+  var getter = this.getters[name];
+
+  if (getter) {
     try {
       // Update module.exports[name] with the current value so that CommonJS
       // require calls remain consistent with module.importSync.
-      return module.exports[name] =
-        this.getters[name].call(module);
-
-    } catch (e) {
-      // If the getter threw an exception, avoid updating module.exports
-      // and return undefined.
-    }
+      module.exports[name] = getter();
```

Comment:
I feel like `runGetter` should return the gotten value, even if we're not using it currently. I'm aware this isn't completely rational, but it seems like something that might be surprising later.